feat(nimbus): Assign tags

yashikakhurana opened the pull request:
Because

- We want to allow users to assign/unassign tags related to the deliveries
- Admin users should be allowed to manage tags in admin panel
This commit

- Ability to assign and unassign tag
- Ability to see the assigned tag near the QA status tile
- Register tag model in admin panel

Fixes #13597

yashikakhurana commented on experimenter/experimenter/nimbus_ui/views.py:
```diff
@@ -874,3 +875,16 @@ def post(self, request, *args, **kwargs):
+    def post(self, request, *args, **kwargs):
+        experiment = self.get_object()
+        selected_tag_ids = request.POST.getlist("tags")
+        experiment.tags.set(selected_tag_ids)
```
let me move this to form

freshstrangemusic commented:
<img width="793" height="833" alt="image" src="https://github.com/user-attachments/assets/80ef8b74-e151-413c-848f-fd73a9b0470b" />
dark mode looks off

yashikakhurana commented on experimenter/experimenter/nimbus_ui/templates/nimbus_experiments/assign_tags_dropdown.html:
```diff
@@ -0,0 +1,50 @@
+              max-height:50vh;
+              overflow:auto">
+    <strong class="dropdown-header">Assign Tags</strong>
+    <form method="post"
```
its a first step towards that, in a following PR will do the improvement to update component only

yashikakhurana commented:
okay @jaredlockhart @freshstrangemusic couple of things, added the admin too so it is easy to test, moved the db related code in the forms, Updated to not use refresh but use htmx to update using `hx-swap-oob`. Fixed the dark mode issue and alligned the count too 
<img width="296" height="90" alt="Screenshot 2025-10-31 at 9 44 44 AM" src="https://github.com/user-attachments/assets/bbced33c-5880-4060-ae9c-1a57381dbf86" />
<img width="873" height="543" alt="Screenshot 2025-10-31 at 9 44 29 AM" src="https://github.com/user-attachments/assets/ea7aba0f-e36e-4843-8122-b28f18c7182c" />

yashikakhurana commented on experimenter/experimenter/nimbus_ui/templates/nimbus_experiments/experiment_tags.html:
```diff
@@ -0,0 +1,16 @@
+        <span class="badge"
+              style="background-color: {{ tag.color }};
+                     color: white;
+                     font-size: 0.75rem">{{ tag.name }}</span>
```
will fix it up on follow-up pr https://github.com/mozilla/experimenter/pull/13848
